stacks-signer/src/chainstate.rs

```diff
@@ -20,14 +20,15 @@ use blockstack_lib::chainstate::stacks::TenureChangePayload;
 use blockstack_lib::net::api::getsortition::SortitionInfo;
 use blockstack_lib::util_lib::db::Error as DBError;
 use clarity::types::chainstate::BurnchainHeaderHash;
+use clarity::util::get_epoch_time_secs;
```

Comment:
this should be `stacks_common::util`

Reply:
Also, the above should be `stacks_common::types::chainstate::BurnchainHeaderHash`.  Clarity merely re-exports these.